ui/src/actions/remoteAccess.js

```diff
@@ -32,11 +32,17 @@ export function updateNickname(name) {
 }
 
 export function requestControl(message) {
-  return () => sendRequestControl(message);
+  return async (dispatch) => {
+    await sendRequestControl(message);
+    dispatch(getLoginInfo());
```

Comment:
As explained, we no longer rely on a socket even to update the `user` state; we refetch it right away.

Reply:
Ok, nice, easier to understand like this